content/sensu-go/5.16/reference/backend.md

```diff
@@ -605,6 +609,34 @@ sensu-backend start --etcd-client-cert-auth
 etcd-client-cert-auth: true{{< /highlight >}}
 
 
+| etcd-discovery        |      |
+------------------------|------
+description             | Exposes [etcd's embedded auto-discovery features][17]. Attempts to use [etcd discovery][18] to get the cluster configuration.
+type                    | String
+default                 | ""
+example                 | {{< highlight shell >}}# Command line example
+sensu-backend start --etcd-discovery https://10.0.0.1:2380
+
+# /etc/sensu/backend.yml example
+etcd-discovery:
+  - https://10.0.0.1:2380
```

Comment:
For this discovery method, I think you should reuse one of the example URLs used in the Etcd docs, e.g.: `https://discovery.etcd.io/3e86b59982e49066c5d813af1c2e2579cbf573de`.